docs/docs/gesture-handlers/about-handlers.md

```diff
@@ -6,7 +6,7 @@ sidebar_position: 1
 ---
 
 :::warning
-Consider using the new [gestures API](/docs/gestures/gesture) instead. The old API is not actively supported and is not receiving the new features. Check out [RNGH 2.0 section in Introduction](/docs/#rngh-20) for more information.
+The old API will be removed in the future version of Gesture Handler. Please migrate to [gestures API](/docs/gestures/gesture) instead. Check out [RNGH 2.0 section in Introduction](/docs/#rngh-20) for more information.
```

Comment:
I think I just removed that section in introduction 🙈. Though, IMO it should be linking to https://docs.swmansion.com/react-native-gesture-handler/docs/guides/upgrading-to-2.

(applies to all)